resources/scripts/ext.networknotice.Notice.js

```diff
@@ -47,11 +47,11 @@
 	/**
 	 * Check on document readyState
 	 */
-	if ( document.readyState === 'complete' ) {
+	if ( document.readyState === 'interactive' ) {
```

Comment:
```suggestion
	if ( ['interactive', 'complete'].includes(document.readyState) ) {
```

Reply:
I tested it on my wikidev but the readystate can also be 'complete' sometimes. So I think it's best to add both?

Reply:
Maybe just this then?
```suggestion
	if ( document.readyState !== 'loading' ) {
```

Reply:
I probably prefer liquidelys version because it stays the same even if another state gets introduced down the line. Would have to be cleaned up for code style of course.

Reply:
updated suggestion for code style